### PR TITLE
Update keras2onnx opset support information

### PR DESCRIPTION
**Pull request by vinitra**
•	All keras layers have been supported since opset 7. Please refer to Keras documentation for all layers. 
•	For Windows ML users, please refer to Windows ML documentation for opset chosen.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,22 +5,24 @@
...
+Initially, the Keras converter was developed in the project [onnxmltools](https://github.com/onnx/onnxmltools). [keras2onnx](https://github.com/onnx/keras-onnx) converter development was moved into a independent repository to support more kinds of Keras models and reduce the complexity of mixing multiple converters.
 
-keras2onnx converter supports not only all keras built-in layers, but also the lambda/custom layer by working with the [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which is embedded into the source tree directly now to avoid version conflicts and installation complexity.
+All Keras layers have been supported for conversion using keras2onnx since ONNX opset 7. Please refer to the [Keras documentation](https://keras.io/layers/about-keras-layers/) for details on Keras layers. The keras2onnx converter also supports the lambda/custom layer by working with the [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which is embedded directly into the source tree to avoid version conflicts and installation complexity.
```
Comment (jiafatom) [inline README.md]: "opset 7" -- put bold to highlight?

File: README.md [hunk 2]
```diff
@@ -5,22 +5,24 @@
 | [![Build Status](https://dev.azure.com/onnxmltools/ketone/_apis/build/status/linux-conda-ci?branchName=master)](https://dev.azure.com/onnxmltools/ketone/_build/latest?definitionId=9&branchName=master) | [![Build Status](https://dev.azure.com/onnxmltools/ketone/_apis/build/status/win32-conda-ci?branchName=master)](https://dev.azure.com/onnxmltools/ketone/_build/latest?definitionId=10&branchName=master) |
 
 
-# Introduction 
+# Introduction
 The keras2onnx model converter enables users to convert Keras models into the [ONNX](https://onnx.ai) model format.
-Initially, the Keras converter was developed in the project [onnxmltools](https://github.com/onnx/onnxmltools). To support more kinds of Keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the Keras model only.
+Initially, the Keras converter was developed in the project [onnxmltools](https://github.com/onnx/onnxmltools). [keras2onnx](https://github.com/onnx/keras-onnx) converter development was moved into a independent repository to support more kinds of Keras models and reduce the complexity of mixing multiple converters.
```
[the remark this answers is not in the document]
Comment (vinitra) [reply README.md]: updated! "independent repository"

File: README.md (hunk 1 above)
Comment (vinitra) [reply README.md]: updated!

File: README.md [hunk 3]
```diff
@@ -5,22 +5,24 @@
...
 
-keras2onnx converter supports not only all keras built-in layers, but also the lambda/custom layer by working with the [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which is embedded into the source tree directly now to avoid version conflicts and installation complexity.
+All Keras layers have been supported for conversion using keras2onnx since **ONNX opset 7**. Please refer to the [Keras documentation](https://keras.io/layers/about-keras-layers/) for details on Keras layers. The keras2onnx converter also supports the lambda/custom layer by working with the [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which is embedded directly into the source tree to avoid version conflicts and installation complexity.
+
+For Windows Machine Learning (WinML) users, please refer to the [WinML Release Notes](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes) to identify the corresponding ONNX opset for your WinML version.
```
Comment (wenbingl) [inline README.md]: For Windows User, you can direct them to user winmltools in which the opset number will be decided by the tool itself.

Comment (vinitra) [reply README.md]: Added a line :)

**Pull request merged**